SPEC.md

```
@@ -0,0 +1,228 @@
+npm run lint:fix
+npm run test -- --coverage
+npx semantic-release   # CI only; local dry-run per plugin docs
+```
+
+Fix legacy script drift (e.g. missing `test:nyc`).
+
+---
+
+## Project structure (target)
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)